trunk/SUAVE/Analyses/Aerodynamics/Vortex_Lattice.py

```diff
@@ -129,6 +133,9 @@ def sample_training(self):
         
         AoA = training.angle_of_attack
         CL  = np.zeros_like(AoA)
+        wing_CLs = Data() 
```

Comment:
I do:
wing_CLs = Data.fromkeys(geometry.wings.keys(), np.zeros_like(AoA))

You can also get rid of the instantiation of wing_CLs as Data. This should be fast because it's written in C.